src/style.css

```diff
@@ -0,0 +1,27 @@
+.blocks {
+  /*display: block;*/
+  /*width: 100%;*/
+}
+
+#block_1 {
+  content: " ";
+  height: 100px;
+  background-color: red;
+}
+
+#block_2 {
+  content: " ";
+  height: 100px;
+  background-color: green;
+}
+
+#block_3 {
+  content: " ";
+  height: 100px;
+  background-color: blue;
+}
```

Comment:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes.  [style guide](https://mate-academy.github.io/style-guides/htmlcss.html#id-selectors)

Reply:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.